src/main/java/com/johan/blignaut/entelect/vitalityincubator/task2/helpers/GeoDBMappingHelper.java

```diff
@@ -0,0 +1,34 @@
+package com.johan.blignaut.entelect.vitalityincubator.task2.helpers;
+
+import com.johan.blignaut.entelect.vitalityincubator.restclient.api.GeodbApi;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.model.GeoDBClient;
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.GeoDBServer;
+import com.johan.blignaut.entelect.vitalityincubator.task2.exception.CustomInternalServerException;
+import com.johan.blignaut.entelect.vitalityincubator.task2.service.GeoDBMappingService;
+import org.springframework.stereotype.Component;
+import retrofit2.Call;
+import retrofit2.Response;
+
+import java.io.IOException;
+
+@Component
+public class GeoDBMappingHelper {
```

Comment:
Since this class is just doing the service call now, and no mapping logic, you could rename it to `GeoDBServiceWrapper` or something like that, in line with Vince's comment above.

To separate the wrapper and mapper even more, you could call the mapper method in the controller class rather than in the service wrapper